pysatSpaceWeather/instruments/sw_f107.py

```diff
@@ -487,8 +479,8 @@ def download(date_array, tag, inst_id, data_path, update_files=False):
                     # Create a local CSV file
                     data.to_csv(data_file, header=True)
     elif tag == 'prelim':
-        ftp = ftplib.FTP('ftp.swpc.noaa.gov')  # connect to host, default port
-        ftp.login()  # user anonymous, passwd anonymous@
+        ftp = ftplib.FTP('ftp.swpc.noaa.gov')  # Connect to host, default port
+        ftp.login()  # User anonymous, passwd anonymous
```

Comment:
```suggestion
        # Connect to host, default port
        ftp = ftplib.FTP('ftp.swpc.noaa.gov')  
        
        # User anonymous, passwd anonymous
        ftp.login()  
```

Reply:
I don't think there's anything wrong with inline comments, do we advise against them in the style guide?